tests/specs/scoped.js

```diff
@@ -0,0 +1,156 @@
+var merge = require('lodash.merge')
+var test = require('tape')
+
+var Store = require('../../')
+var options = process.browser ? {
+  adapter: 'memory'
+} : {
+  db: require('memdown')
+}
+
+test('scoped method', function (t) {
+  t.plan(1)
+
+  var store = new Store('test-db-scoped', merge({remote: 'test-db-scoped'}, options))
+
+  t.is(typeof store.scoped, 'function', 'has "scoped" method')
+})
+
+test('store.scoped("test") with methods', function (t) {
+  t.plan(12)
+
+  var store = new Store('test-db-scoped-store', merge({remote: 'test-db-scoped-store'}, options))
+  var testStore = store.scoped('test')
+
+  t.is(typeof testStore.add, 'function', 'has "add" method')
+  t.is(typeof testStore.find, 'function', 'has "find" method')
+  t.is(typeof testStore.findOrAdd, 'function', 'has "findOrAdd" method')
+  t.is(typeof testStore.findAll, 'function', 'has "findAll" method')
+  t.is(typeof testStore.update, 'function', 'has "update" method')
+  t.is(typeof testStore.updateOrAdd, 'function', 'has "updateOrAdd" method')
+  t.is(typeof testStore.updateAll, 'function', 'has "updateAll" method')
+  t.is(typeof testStore.remove, 'function', 'has "remove" method')
+  t.is(typeof testStore.removeAll, 'function', 'has "removeAll" method')
+  t.is(typeof testStore.on, 'function', 'has "on" method')
+  t.is(typeof testStore.one, 'function', 'has "one" method')
+  t.is(typeof testStore.off, 'function', 'has "off" method')
+})
+
+test('scoped Store .on("change") with adding one', function (t) {
+  t.plan(4)
+
+  var store = new Store('test-db-scoped-add', merge({remote: 'test-db-scoped-add'}, options))
+  var testStore = store.scoped('test')
+  var changeEvents = []
+
+  testStore.on('change', addEventToArray.bind(null, changeEvents))
+  store.on('change', addEventToArray.bind(null, changeEvents))
+
+  testStore.add({
+    foo: 'bar'
+  })
+
+  .then(function () {
+    t.is(changeEvents.length, 1, 'triggers 1 change event')
+    t.is(changeEvents[0].eventName, 'add', 'passes the event name')
+    t.is(changeEvents[0].object.foo, 'bar', 'event passes object')
+    t.is(changeEvents[0].object.type, 'test', 'event passes object with scoped type')
+  })
+})
```

Comment:
Ah ok, I thought the event was completely scoped to that new `store`. That makes this a little easier. :smile: